packages/react-cosmos-router-proxy/src/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { MemoryRouter, Route } from 'react-router';
+import proxyPropTypes from 'react-cosmos-utils/lib/proxy-prop-types';
+import LocationInterceptor from './LocationInterceptor';
+
+export default () => {
+  const RouterProxy = props => {
+    const { nextProxy, fixture, onFixtureUpdate } = props;
+    const { value: NextProxy, next } = nextProxy;
+    const { route, url } = fixture;
+
+    const children = (
+      <LocationInterceptor onLocation={url => onFixtureUpdate({ url })}>
```

Comment:
I think there's some context that I'm missing but what is onFixtureUpdate's role?